src/middlewared/middlewared/plugins/acme_protocol_/cloudflare.py

```diff
@@ -0,0 +1,106 @@
+from certbot.plugins import dns_common
+from CloudFlare.cloudflare import CloudFlare, CloudFlareAPIError
```

Comment:
Do we already have that package in debian control?

Reply:
Yup